### PR TITLE
feat: add login button to guest welcome window

### LINKED ISSUE #12772: Ease participating as a internal user when joining public conversations
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Feature request

**Which Nextcloud Version are you currently using:** 29

**Is your feature request related to a problem? Please describe.**
As a user of talk I want to be able to join a conversation, with my internal user, even if I am not currently logged in, because there is just a manual way to achive this.

After opening to a conversatiin with a public link, I only have the option to enter a name for my current guest user. If I want to join with my internal identity, I have to enter the URL of the login page, enter my credentials. After that I have to change to the talk app and search the conversation I want to join. Or I have to reopen my mail program to fetch the link of the invitation. 

**Describe the solution you'd like**
Add an option to login to the welcome screen. Either by allowing to use the login there or route to the login page and rediret to the conversation.

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

* Fix #12772

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | image
Lobby | image
Sidebar | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

**Review by ShGKme — DISMISSED** [review]

File: src/components/GuestWelcomeWindow.vue [hunk 1]
```diff
@@ -35,6 +35,14 @@
 					<Check :size="20" />
 				</template>
 			</NcButton>
+
+			<div class="login-info">
+				<span> {{ t('spreed', 'Do you already have an account ?') }}</span>
```
Comment (nickvergessen) [inline src/components/GuestWelcomeWindow.vue:40]:
```suggestion
				<span> {{ t('spreed', 'Do you already have an account?') }}</span>
```

File: src/components/GuestWelcomeWindow.vue [hunk 2]
```diff
@@ -137,4 +150,11 @@ export default {
 .submit-button {
 	margin: 0 auto;
 }
+
+.login-info{
```
Comment (nickvergessen) [inline src/components/GuestWelcomeWindow.vue:154]:
```suggestion
.login-info {
```

**Comment by nickvergessen** [comment]
We should also have the button in the right sidebar, so that you can login even when your browser storage has a name already?

File: src/components/GuestWelcomeWindow.vue [hunk 3]
```diff
@@ -35,6 +35,14 @@
 					<Check :size="20" />
 				</template>
 			</NcButton>
+
+			<div class="login-info">
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/GuestWelcomeWindow.vue:39]: like this 
image

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**